### DISCUSSION TITLE
Alternate existing API to be used instead of druid service call API in course service

### DISCUSSION THREAD

**Discussion by BharathwajShankar** [Sunbird-Obsrv/Community#54] [Query]
Hi Team,
            In the lms service, we are currently using druid query on "audit-rollup-syncts" and "telemetry-events-syncts"  datasources for fetching collection summary aggregate data. Kindly suggest to us if there is any existing API that can be used to get this data without making a druid call. @anandp504 @manjudr cc: @rhwarrier @reshmi-nair

**Comment by anandp504** [comment]
@BharathwajShankar @reshmi-nair We should not be querying from the "telemetry-events-syncts" datasource as it contains huge amount of data. We will be deploying the [obsrv-api-service](https://github.com/Sunbird-Obsrv/obsrv-api-service) as part of the release 5.2.0. Also, can you please explain what is the collection summary aggregate data is fetched from audit-rollup-syncts data source?

**Reply by BharathwajShankar** [reply]
@anandp504 we use audit-rollup-syncts data source to get enrolment and completion counts location-wise.

**Reply by BharathwajShankar** [reply]
@anandp504  can you please share the API specification details here?
cc: @reshmi-nair

**Reply by manjudr** [reply]
@BharathwajShankar  - The API Swagger documentation link - https://github.com/Sanketika-Obsrv/obsrv-api-service/blob/main/swagger-doc/openapi.yaml.  We will be deploying as part of the release 5.2.0.

**Reply by reshmi-nair** [reply]
@manjudr Will the [obsrv-api-service](https://github.com/Sunbird-Obsrv/obsrv-api-service)  get released as part of 5.3.0 sprint 1? Please confirm on this. Lern has some dependency tickets that needs to be planned for 5.3.0 sprint 2.
cc @anandp504 @rhwarrier

**Reply by BharathwajShankar** [reply]
@manjudr any updates on the above query of Reshmi? Will the [obsrv-api-service](https://github.com/Sunbird-Obsrv/obsrv-api-service) get released as part of 5.3.0 sprint 1? Please confirm this. Lern has some dependency tickets that need to be planned for 5.3.0 sprint 2.

cc @anandp504 @rhwarrier

**Comment by rhwarrier** [comment]
Team - please call out an ETA if known so that the Lern team can plan accordingly. Thanks

**Reply by BharathwajShankar** [reply]
@manjudr any luck on the ETA? Lern has some dependency tickets that need to be planned for 5.3.0 sprint 2 which starts from tomorrow.

cc @rhwarrier @reshmi-nair @anandp504
